django/db/backends/utils.py

```diff
@@ -65,14 +66,26 @@ def callproc(self, procname, params=None, kparams=None):
                 return self.cursor.callproc(procname, params, kparams)
 
     def execute(self, sql, params=None):
+        return self._execute_with_wrappers(sql, params, False, self._execute)
```

Comment:
It would be more readable to pass boolean arguments as keywords, e.g. `many=False`.

Reply:
Added.